### ISSUE TITLE
Remove `xyz` tiledir from outputs once used to bake mbtiles

### ISSUE THREAD

**Issue by rudokemper** [ConservationMetrics/guardianconnector-change-detection#6]
Once step 6 (Convert raster XYZ directory to MBTiles) is complete, the `xyz/` tiledir in outputs is no longer needed. The case can be made that there is value in keeping them for usage in the old Mapeo map server, but since that is soon to be replaced by the new map server, and we're dependent on using the new map server for vector mbtiles anyway, this seems moot.

This is a quick fix which I'll take care of next week.
